src/Magnum/DartIntegration/Object.h

```diff
@@ -40,72 +45,93 @@ namespace dart { namespace dynamics {
     class ShapeNode;
 }}
 
-namespace Magnum { namespace DartIntegration {
+namespace Magnum {
+
+class Mesh;
+class Buffer;
+template<UnsignedInt dimensions> class Texture;
+
+namespace DartIntegration {
+
+/**
+@brief Shape data
+
+@see @ref Object::_convertShapeNode()
+*/
+struct ShapeData {
+    /** @brief Mesh */
+    /* @todo: support multiple meshes */
+    Mesh* mesh;
+
+    /** @brief vertex Buffer */
+    Buffer* vertexBuffer;
+
+    /** @brief index Buffer */
+    Buffer* indexBuffer;
+
+    /** @brief Material data */
+    /* @todo: support multiple materials */
+    Trade::PhongMaterialData material;
+
+    /** @brief Textures */
+    Containers::Array<Texture<2>*> textures;
+};
 
 /**
 @brief DART Physics BodyNode or ShapeNode
 
 Encapsulates `BodyNode` or `ShapeNode` as a @ref SceneGraph feature.
 
-@section DartIntegration-DartObject-usage Usage
+@section DartIntegration-Object-usage Usage
 
-Common usage is to create a @ref DartObject to share transformation with a DART
+Common usage is to create a @ref Object to share transformation with a DART
 `BodyNode` or `ShapeNode` by passing a pointer to its constructor:
 
 @code{.cpp}
 dart::dynamics::BodyNode* body = getBodyNodeFromDart();
 SceneGraph::Object<SceneGraph::MatrixTransformation3D> object;
-DartObject* obj = new DartObject{&object, body};
+Object* obj = new Object{&object, body};
 @endcode
 
 or
 
 @code{.cpp}
 dart::dynamics::ShapeNode* node = getShapeNodeFromDart();
 SceneGraph::Object<SceneGraph::MatrixTransformation3D> object;
-DartObject* obj = new DartObject{&object, node};
+Object* obj = new Object{&object, node};
```

Comment:
After this rename I would prefer to use the explicit namespace in the documentation snippets (`DartIntegration::Object`), similarly to what's done with SceneGraph APIs.

Reply:
> After this rename I would prefer to use the explicit namespace in the documentation snippets (DartIntegration::Object), similarly to what's done with SceneGraph APIs.

Good point!